config.json

```diff
@@ -1117,6 +1117,18 @@
       "unlocked_by": "triangle",
       "uuid": "52d11278-0d65-4b5b-b387-1374fced3243"
     },
+    {
+      "core": false,
+      "difficulty": 8,
+      "slug": "rational-numbers",
+      "topics": [
+        "floating_point_numbers",
+        "mathematics",
+        "classes"
```

Comment:
What is your reasoning for `classes`?

If you couldn't find it (I always have trouble locating it) the [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) is super useful here!

Reply:
@sjwarner-bp I saw that the topics of complex-numbers, which is similar to this problem, has classes. That's all. I'll take a look at that list.

Reply:
It's not the easiest list to find! :laughing: Your suggestions seem sensible so far, but feel free to add anything else you think might be suitable!